lib/engine/step/special_track.rb

```diff
@@ -78,6 +82,19 @@ def check_connect(action)
       def setup
         @company = nil
       end
+
+      private
+
+      def allowed_to_lay_tile?(entity)
+        # If corporation owns the company, this is allowed
+        return true if @game.current_entity.corporation? && entity.owner == @game.current_entity
+
+        # This is if tile lay is a continuation of a special tile lay
+        return true if @game.current_entity == entity
+
+        # Player owned company can be used
+        ability(entity).owner_type == :player && entity.owner == @game.current_entity.player
```

Comment:
this breaks mitsubishi ferry which can be laid anytime

Reply:
Ok, so can we figure out who made the special tile lay? If not just the active player can do it?  

Or is 1284 not possible to solve?

This could be a case of something the players have to police themselves. In that case we can just update the wiki with this and let it be.

Reply:
Mitsubishi Ferry cannot be laid during an OR turn of a corporation of another President. 18xx.games wording:

> Player owner may place the port tile on a coastal town (B11, G10, I12, or J9) without a tile on it already, outside of the operating rounds of a corporation controlled by another player.